<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OBSPermissions</class>
 <widget class="QDialog" name="OBSPermissions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>692</width>
    <height>613</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MacPermissions.Title</string>
  </property>
  <property name="windowIcon">
   <iconset resource="obs.qrc">
    <normaloff>:/res/images/obs.png</normaloff>:/res/images/obs.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>MacPermissions.Description</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="0">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="3" column="0">
          <widget class="QPushButton" name="continueButton">
           <property name="text">
            <string>MacPermissions.Continue</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>MacPermissions.Description.OpenDialog</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
           <property name="bottomMargin" stdset="0">
            <number>8</number>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <layout class="QFormLayout" name="formLayout">
           <property name="fieldGrowthPolicy">
            <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
           </property>
           <property name="rowWrapPolicy">
            <enum>QFormLayout::WrapAllRows</enum>
           </property>
           <property name="topMargin">
            <number>22</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="label_4">
             <property name="font">
              <font>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>MacPermissions.Item.ScreenRecording</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>MacPermissions.Item.ScreenRecording.Details</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="topMargin">
              <number>8</number>
             </property>
             <property name="bottomMargin">
              <number>25</number>
             </property>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="capturePermissionButton">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>300</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>300</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="baseSize">
                <size>
                 <width>300</width>
                 <height>0</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_7">
             <property name="font">
              <font>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>MacPermissions.Item.Camera</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>MacPermissions.Item.Camera.Details</string>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="topMargin">
              <number>8</number>
             </property>
             <property name="bottomMargin">
              <number>25</number>
             </property>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="videoPermissionButton">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>300</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>300</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="baseSize">
                <size>
                 <width>300</width>
                 <height>0</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_9">
             <property name="font">
              <font>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>MacPermissions.Item.Microphone</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>MacPermissions.Item.Microphone.Details</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <property name="topMargin">
              <number>8</number>
             </property>
             <property name="bottomMargin">
              <number>25</number>
             </property>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="audioPermissionButton">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>300</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>300</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="baseSize">
                <size>
                 <width>300</width>
                 <height>0</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="label_3">
             <property name="font">
              <font>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>MacPermissions.Item.InputMonitoring</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>MacPermissions.Item.InputMonitoring.Details</string>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="7" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <property name="sizeConstraint">
              <enum>QLayout::SetDefaultConstraint</enum>
             </property>
             <item>
              <widget class="QWidget"/>
             </item>
             <item>
              <widget class="QPushButton" name="inputMonitoringPermissionButton">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>300</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>300</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="baseSize">
                <size>
                 <width>300</width>
                 <height>0</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="8" column="1">
            <widget class="QLabel" name="accessibilityPermissionLabel">
             <property name="text">
              <string>MacPermissions.Item.Accessibility.Details</string>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="9" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <property name="sizeConstraint">
              <enum>QLayout::SetDefaultConstraint</enum>
             </property>
             <property name="topMargin">
              <number>8</number>
             </property>
             <property name="bottomMargin">
              <number>25</number>
             </property>
             <item>
              <spacer name="horizontalSpacer_4">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="accessibilityPermissionButton">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>300</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>300</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="baseSize">
                <size>
                 <width>300</width>
                 <height>0</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="obs.qrc"/>
 </resources>
 <connections/>
</ui>
